drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -599,10 +605,48 @@ def self.staff_assignment_relationships(project)
 
     def self.projects_receiving_referrals(data_source_id)
       Hmis::Hud::Project.where(data_source_id: data_source_id).
-        receiving_referrals.
+        receiving_legacy_referrals.
         joins(:organization).preload(:organization).
         sort_by_option(:organization_and_name).
         map(&:to_pick_list_option)
     end
+
+    def self.projects_receiving_direct_ce_referrals(from_project:)
+      return [] unless Hmis::Ce.configuration.enabled?
+      return [] unless Hmis::ProjectConfig.with_config_type('COORDINATED_ENTRY').any?
+
+      # Load all projects in the data source into memory and iterate through them to call detect_best_config_for_project.
+      project_scope = Hmis::Hud::Project.where(data_source: from_project.data_source).
+        preload(:organization).
```

Comment:
Lets remove the ability to refer to the same project:
```suggestion
        where.not(id: from_project.id).
        preload(:organization).
```

Reply:
Thanks!